rocAL/include/pipeline/commons.h

```diff
@@ -239,6 +250,7 @@ enum class ResizeInterpolationType {
     GAUSSIAN,
     TRIANGULAR
 };
+REGISTER_ENUM(ResizeInterpolationType)
```

Comment:
Non-blocking: Associated enum is defined as . Consider explicitly specifying the underlying type (e.g., ) to keep ABI stable and avoid surprises across compilers.